client/src/services/webhooks.js

```diff
@@ -1,49 +0,0 @@
-import React, {
```

Comment:
I wonder why we don't need this anymore 🤔

Reply:
we used to hit an endpoint to get the webhook to pass in link. since we do it server side we no longer need it